tools/modules/software/module_apt_browsers.sh

```
@@ -0,0 +1,60 @@
+module_options+=(
+	["_checklist_browsers,author"]="@Tearran"
+	["_checklist_browsers,maintainer"]="@Tearran"
+	["_checklist_browsers,feature"]="_checklist_browsers"
+	["_checklist_browsers,example"]=""
+	["_checklist_browsers,desc"]="Browser installation and management (Firefox-ESR and Chromium and more)."
+	["_checklist_browsers,status"]="Active"
+	["_checklist_browsers,group"]="Internet"
+	["_checklist_browsers,arch"]="x86-64 arm64 armhf"
+)
+# Scaffold for app with specific, single, dummy or other candidates.
+function _checklist_browsers() {
```

Shellcheck (0.10.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)